src/java/org/apache/cassandra/cql3/ResultSet.java

```diff
@@ -47,7 +47,7 @@ public class ResultSet
     public static final Codec codec = new Codec();
 
     public final ResultMetadata metadata;
-    public final List<List<ByteBuffer>> rows;
+    public List<List<ByteBuffer>> rows;
```

Comment:
Nit: I think we can keep this as a `final` variable.

Reply:
Thanks - I addressed it [here](https://github.com/datastax/cassandra/pull/710/commits/fff4a3327f31f2a812bd8bc77472ba423005c860).